components/squad/squad_row.lua

```diff
@@ -72,19 +84,25 @@ function SquadRow:role(args)
 	end
 
 	self.content:node(cell)
+
+	self.lpdbData['role'] = args.role
+
 	return self
 end
 
-function SquadRow:date(dateValue, mobileTitle)
+function SquadRow:date(dateValue, cellTitle, lpdbColumn)
 	local cell = mw.html.create('td')
 	cell:addClass('Date')
 
-	cell:node(mw.html.create('div'):addClass('MobileStuffDate'):wikitext(mobileTitle))
+	cell:node(mw.html.create('div'):addClass('MobileStuffDate'):wikitext(cellTitle))
 
 	if not String.isEmpty(dateValue) then
 		cell:node(mw.html.create('div'):addClass('Date'):wikitext('\'\'' .. dateValue .. '\'\''))
 	end
 	self.content:node(cell)
+
+	self.lpdbData[lpdbColumn] = ReferenceCleaner.clean(dateValue)
+
```

Comment:
since lpdb has a joindaterred/leavedateref/inactiveref field, maybe we also should store the ref here?